server.js

```diff
@@ -4618,6 +4618,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CRAN/METACRAN integration.
+camp.route(/^\/cran\/([^\/])\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var info = match[1]; // either `v` or `l`
+  var pkg = match[2]; // eg, devtools
+  var format = match[3];
+  var url = 'http://crandb.r-pkg.org//' + pkg;
```

Comment:
Im pretty sure there was a reason for the double slash the time I made the PR. I just checked and either link works to get the JSON.